python/common/org/python/types/Str.java

```diff
@@ -764,7 +764,17 @@ private long toPositiveIndex(long index) {
             __doc__ = ""
     )
     public org.python.Object isnumeric() {
-        throw new org.python.exceptions.NotImplementedError("isnumeric() has not been implemented.");
+        int c = 0;
+        java.lang.String checkString = this.value;
+        char currentCharacter;
+        while (c < (checkString.length()-1)) {
```

Comment:
There are some style checks failing in the build: https://travis-ci.org/pybee/voc/jobs/210692217#L187-L188

Also, this looks like there is an off-by-one error, you should add a test for a string like `3.14M`.